server/controllers/userController.js

```diff
@@ -17,8 +18,8 @@ class UsersController {
  * @returns {object} - returns user
  */
   static async signUp(req, res) {
-    const { email, first_name, last_name } = req.body;
-    let { password } = req.body;
+    const { email, first_name, last_name } = toLowerCase(req.body);
+    let { password } = toLowerCase(req.body);
```

Comment:
Hey is there a reason to make the password case insensitive?

Reply:
That was a mistake on my end. Thanks for pointing out this.